code-base/src/main.py

```diff
@@ -1,58 +1,49 @@
 import os
 import click
 
-project_types = ["basic", "web", "react", "angular", "gatsby"]
-
-
 # Mainloop
 @click.command()
 @click.argument('input')
 @click.option('--type', '-t', 'project_type')
 @click.option('--name', '-n', 'name')
 def main(input, project_type, name): 
     input = input.lower()
-    if input == "create" and check_project_type(project_type.lower()):
+    project_type = project_type.lower()
+    if input == "create" and check_project_type(project_type):
         if(project_type == "basic"):
             basic_project(name)
     elif input == "prompt":
-        titleScreen()
-        askForProjectName()
+        title_screen()
+        ask_for_project_name()
     else:
         print("Invalid flags!")
 
 
-
 # Helper Functions
 # ------------------------------------------------------------------
 
 """First screen the user will see"""
-def titleScreen():
+def title_screen():
```

Comment:
[Type hinting](https://docs.python.org/3/library/typing.html) for your functions. can be pretty helpful.

Reply:
nice, reminds me of Haskell. Is it actually useful though? Python is dynamically typed after all.

Reply:
> nice, reminds me of Haskell. Is it actually useful though? Python is dynamically typed after all.

Due to the dynamic nature of Python, inferring, or checking the type of an object being used is especially hard. With type hinting, you hint the type of the object(s) you're using. If you are more curious you can check this presentation here: https://www.youtube.com/watch?v=2wDvzy6Hgxg